packages/components/src/LegacyHeader/LegacyMobileHeader.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled from 'styled-components';
+import { color, layout, space } from 'styled-system';
+
+import { ThemeProps } from '@t3n/theme';
+
+import { Box } from '../Box';
+import { Logo } from '../Logo';
+import { Image } from '../Image';
+import { LegacyUserMenuProps } from '../LegacyUserMenu';
+import { LegacyMobileNav } from './components/LegacyMobileNav';
+import { HeaderCampaign } from './LegacyHeader';
+
+const MobileHeader = styled(Box)`
+  width: 100%;
+  ${({ theme }) =>
+    color({ theme, bg: 'shades.white', color: 'text.secondary' })};
```

Comment:
Hintegründe bitte per `background`-Property referenzieren, in dem Fall also `background.primary`